source/views/components/cells/multi-line-detail.js

```diff
@@ -20,13 +20,15 @@ export class MultiLineDetailCell extends React.PureComponent<Props> {
 					<Text allowFontScaling={true} style={styles.cellTitle}>
 						{title}
 					</Text>
-					<Text allowFontScaling={true} style={styles.cellLeftDetail}>
-						{leftDetail}
-					</Text>
+					{this.props.leftDetail && (
+						<Text allowFontScaling={true} style={styles.cellLeftDetail}>
+							{leftDetail}
+						</Text>
+					)}
 				</View>
-				<Text allowFontScaling={true} style={styles.cellRightDetail}>
+				<View allowFontScaling={true} style={styles.cellRightDetail}>
```

Comment:
Changing from `Text` to `View` means we'll also need to change the style. We can't apply`letterSpacing` to a `View`.